sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemRequestOptions.java

```diff
@@ -319,4 +322,24 @@ public String getThroughputControlGroupName() {
     public void setThroughputControlGroupName(String throughputControlGroupName) {
         this.throughputControlGroupName = throughputControlGroupName;
     }
+
+    /**
+     * Gets the thresholdForDiagnosticsOnTracerInMS, if latency on CRUD operation is greater than this
+     * diagnostics will be send to open telemetry exporter as events in tracer span of end to end CRUD api.
+     *
+     * @return  thresholdForDiagnosticsOnTracerInMS the latency threshold for diagnostics on tracer.
+     */
+    public int getThresholdForDiagnosticsOnTracerInMS() {
```

Comment:
we usually use Duration in public API for representing time period. can we use the same thing?

Reply:
done